src/test/java/org/opentripplanner/api/parameter/QualifiedModeSetTest.java

```diff
@@ -211,4 +215,24 @@ void specificallyRequestCarpool() {
 
     assertEquals(mainModes, expected);
   }
+
+  @Test
+  void carHail() {
+    var modeSet = new QualifiedModeSet("CAR_HAIL");
```

Comment:
I do but it will open up a big can of worms: it's BICYCLE_RENT at the API level but BIKE_RENTAL internally. Therefore I went with the same pattern even though it's a bit confusing.